omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/DeletePatientFragmentControllerTest.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.coreapps.fragment.controller;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.Patient;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.context.Context;
+
+public class DeletePatientFragmentControllerTest {
+	
+	@Test
+	public void shouldDeletePatientOnConfirm() throws Exception {
+		DeletePatientFragmentController controller = new DeletePatientFragmentController();
+		//PatientService patientService = Context.getPatientService();
```

Comment:
Is there any unit test I can refer to? There isn't a single relevant unit test in the coreapps module. Plus the style of writing unit tests is completely different from the old core stuff I have done. I'll try and look it up in the wiki/other module meanwhile